config/default/test/targets/pregnancy.spec.js

```diff
@@ -43,6 +43,7 @@ describe('Pregnancy related targets test', () => {
         expect(activePregnancies[0]).to.nested.not.include({ 'value.pass': 1 });
         expect(activePregnancies[0]).to.nested.not.include({ 'value.total': 1 });
       }
+      clock.restore();
```

Comment:
Theses were needed because the new version of `sinon` did not like calling `useFakeTimers` multiple times without restoring the previous clock first...